realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1946,7 +1947,7 @@ public boolean isValid() {
     /**
      * Calculates the sum of a given field.
```

Comment:
Sounds like the doc could use some comments on what the actual values are when used on mixed types.